addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -54,69 +54,50 @@
-                                    <span name="ignored_text">emails have been ignored and will not be sent.</span>
-                                </strong>
-                            </button>
-                        </div>
-                        <div attrs="{'invisible': [('scheduled', '=', 0)]}">
-                            <button class="btn-link py-0"
-                                    name="action_view_traces_scheduled"
-                                    type="object">
-                                <strong>
-                                    <field name="scheduled" class="oe_inline mr-2"/>
-                                    <span name="scheduled_text">emails are in queue and will be sent soon.</span>
-                                </strong>
-                            </button>
-                        </div>
-                        <div attrs="{'invisible': ['&amp;', ('sent', '=', 0), ('state', 'in', ('draft', 'test', 'in_queue'))]}">
-                            <button class="btn-link py-0"
-                                    name="action_view_traces_sent"
-                                    type="object">
-                                <strong>
-                                    <field name="sent" class="oe_inline mr-2"/>
-                                    <span name="sent">emails have been sent.</span>
-                                </strong>
-                            </button>
-                        </div>
-                        <div attrs="{'invisible': ['|', ('state', '!=', 'done'), ('failed', '=', 0)]}">
-                            <button class="btn-link py-0"
-                                    name="action_view_traces_failed"
-                                    type="object">
-                                <strong>
-                                    <field name="failed" class="oe_inline mr-2"/>
-                                    <span name="failed_text">emails could not be sent.</span>
-                                </strong>
-                            </button>
-                        </div>
-                        <div attrs="{'invisible': [('state', '!=', 'in_queue')]}">
-                            <strong>
-                                <span name="next_departure_text">This mailing is scheduled for </span>
-                                <field name="next_departure" class="oe_inline"/>.
-                            </strong>
-                        </div>
-                    </div>
+                            <div class="alert alert-info text-center"
+                                role="alert"
+                                attrs="{'invisible': ['&amp;','&amp;',('state', '!=', 'in_queue'),('scheduled', '=', 0),('failed', '=', 0)]}">
+                                <div attrs="{'invisible': ['|', ('state', '!=', 'done'), ('failed', '=', 0)]}">
+                                    <button class="btn-link py-0"
```

Comment:
Linting: Strange, in hereabove div you indent lines, here you align properties. Somehow you will have to choose your styling (we generally go for indentation and avoid properties alignment in standard views). To propagate to other occurrences.